CHANGELOG.md

```diff
@@ -93,6 +93,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `MarqoVectorStoreDriver.query` failing when `include_metadata` is `True`.
 - `with_contextvars` not properly wrapping functions in some cases.
 - Crash when calling `ToolkitTask.run()` directly.
+- Support default values using kwarg injection on `@activity` methods.
```

Comment:
Is this meant to be under Fixed?

Reply:
I think so. currently if you defined a schema with 3 params: `foo` , `bar`, `baz`, and your activity method was defined as:
```
def activity_method(foo: str, bar: str, baz: str = "default"): ...
```
and the input looked like:
```
{
  "values": {
    "foo": "some_foo",
    "bar": "some_bar"
  },
}
```
the decorator would set `baz=None` rather than keeping the default that was defined on the method signature

Reply:
Got it. Can you rephrase the changelog entry with an implicit "Fixed"?